libs/category/state/src/selectors/category.selector.ts

```diff
@@ -23,17 +23,41 @@ import {
   getDaffCategoryPageSelectors,
 } from './category-page/category-page.selector';
 
+/**
+ * An interface for selectors that combine category metadata and category entities selectors.
+ */
 export interface DaffCategoryMemoizedSelectors<
   V extends DaffGenericCategory<V> = DaffCategory,
   W extends DaffProduct = DaffProduct
   > extends
   DaffCategoryFeatureMemoizedSelectors<V>,
   DaffCategoryPageMemoizedSelectors<V>,
   DaffCategoryEntitiesMemoizedSelectors<V> {
+	/**
+	 * Selects the current category
+	 */
   selectSelectedCategory: MemoizedSelector<Record<string, any>, V>;
```

Comment:
Kind of related to this PR because of the docs you wrote — wouldn't it make more sense for this property to be named `selectCurrentCategory`.

Reply:
I think so, yea. I don't think I should make breaking changes in this PR though. I can remove the doc if you'd prefer.

Reply:
I think you can keep the doc, but make an issue about updating the property name?

Reply:
added: https://github.com/graycoreio/daffodil/issues/1648